SdlLogs/SdlLogs.py

```diff
@@ -223,25 +225,61 @@ def run(self, edit):
                 there is value in 'source_path' attribute in settings file
                 there isn't value in 'source_path' attribute in settings file
         """
-        FULL_PATH_FILE = "(/.*)(/sdl_core/src/.{0,}\\.(cc|h|cpp|hpp)):(\\d{1,})"
-        CORRECTED_PATH = "(/[^/ ]*)+/?"
-        primaryPath = re.search(FULL_PATH_FILE, self.view.substr(
-            self.view.line(self.view.sel()[0])))
-        isItPath = re.search(CORRECTED_PATH, self.view.substr(
+
+        def open_file(self, filePath):
+            """
+                The function hides the function call to jump to file and to the desired line.
+            """
+            self.view.window().open_file(filePath, sublime.ENCODED_POSITION)
```

Comment:
@MaximRybak what if there is no such file? I would say the one function that opens file and show error in case of any error, is enough

Reply:
Fixed in c82425a84f05a787910dbe0681d7d4b1f6a15a91